vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -724,23 +768,31 @@ func verifyTransferSubnetOwnershipTx(
 }
 
 // Ensure the proposed validator starts after the current time
-func verifyStakerStartTime(chainTime, stakerTime time.Time) error {
-	if !chainTime.Before(stakerTime) {
-		return fmt.Errorf(
-			"%w: %s >= %s",
-			ErrTimestampNotBeforeStartTime,
-			chainTime,
-			stakerTime,
-		)
+func verifyStakerStartTime(isDurangoActive bool, chainTime, stakerTime time.Time) error {
+	// Pre Durango activation, start time must be after current chain time.
+	// Post Durango activation, start time is not validated
+	if !isDurangoActive {
```

Comment:
Can we invert this if to remove one level of nesting?

```suggestion
	if isDurangoActive {
		return nil
	}
```